doc/api/cli.md

```diff
@@ -25,19 +25,19 @@ _For more info about `node debug`, please see the [debugger][] documentation._
 
 ### `-v`, `--version`
 
-Print node's version.
+Print version number.
 
 
 ### `-h`, `--help`
 
-Print node command line options.
-The output of this option is less detailed than this document.
+Print options.
+The output is less detailed than this document.
 
 
 ### `-e`, `--eval "script"`
 
-Evaluate the following argument as JavaScript. The modules which are
-predefined in the REPL can also be used in `script`.
+Evaluate the argument as JavaScript. The modules which are predefined in the
+REPL can also be used in `script`.
```

Comment:
I don’t know if this change in wording doesn’t make things a bit more confusing, as it sounds to me as if something like `node --eval --no-deprecation process.EventEmitter` would work?